### ISSUE TITLE
Can't hash strings that contain whitespace

### ISSUE THREAD

**Issue by pzelnip** [tgfjt/hubot-hashme#2]
If you pass in content which contains whitespace, the regex breaks:

```
Hubot> hubot hashme sha1 This is some text
Hubot> algorithm should to be... sha1, md5, sha256, sha512
```

**Comment by tgfjt** [comment]
oh:crying_cat_face:
Now I'll fix it ASAP!

**Comment by tgfjt** [comment]
@pzelnip 
Fixed v0.3.0. Thanks!

**Comment by pzelnip** [comment]
Awesome, thanks!
